### DISCUSSION TITLE
How to switch power mode and refresh rate automatically?

### DISCUSSION THREAD

**Discussion by Infiniti151** [seerge/g-helper#2287] [Q&A]
Hi everyone,

I want the refresh rate to switch to 60 Hz and power mode to change to silent on unplugging my G14 and go back to 120 Hz and balanced mode after plugging in. Currently, the refresh rate stays at whichever is selected and power mode stays at balanced. Any way to fix this?

**Comment by seerge** [comment]
@aadish151 set display to `Auto`, it does literally that

**Reply by Infiniti151** [reply]
Thanks! Any way to do that for the power modes as well? Silent on unplugged and balanced on plugged?

**Reply by seerge** [reply]
App remembers last used mode in each situation and switches automatically, please check readme

**Reply by Infiniti151** [reply]
Thanks! That fixed it👍

**Comment by T3chminator** [comment]
Is there a way for Ghelper to auto switch power plan when an application is launched and to revert back to the previous plan after it is closed. I am envisioning a scenario wherein when I launch a game, the power plan automatically switches from silent to balanced and goes back to silent once the game is closed.

**Reply by seerge** [reply]
@T3chminator G-Helper doesn't monitor apps running and launching in your system anyhow. But you can create as many custom modes (including [custom power plans](https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode)) as you want and just turn them on manually, or with a hotkeys 

https://github.com/seerge/g-helper?tab=readme-ov-file#%EF%B8%8F-keybindings
